src/intTest/resources/outbound_uat_data/acceptance/type3-transferin-all.edifact.dat

```diff
@@ -21,8 +21,8 @@ PNA+PAT+N/10/10:OPI+++SU:ASQUITH+FO:ROGER+TI:LORD+MI:ADAM+FS:DANIEL'
 DTM+329:19460723:102'
 PDI+1'
 NAD+PAT++THE MANSION HOUSE:1 MANSION LANE:ST PAULS CRAY:ORPINGTON:KENT+++++BR12 7ET'
-NAD+PER++THE HAUNTED HOUSE:12 SHADOWY LANE:HAUNTED GLADE:HORSFORTH:LEEDS'
 S02+2'
 PNA+PER++++SU:PREVIOUS'
-UNT+21+00000005'
-UNZ+1+00000004'
+NAD+PER++THE HAUNTED HOUSE:12 SHADOWY LANE:HAUNTED GLADE:HORSFORTH:LEEDS'
```

Comment:
This was correct originally, previous address belongs in the first occurrence of S02